storage/store.go

```diff
@@ -556,6 +556,23 @@ func (s *Store) ForceReplicationScan() {
 	}
 }
 
+// WaitForRange waits for the specified range to acquire the
+// leader lease. Exposed only for testing.
+func (s *Store) WaitForRange(raftID int64, t testing.TB) {
```

Comment:
I'd rather not be so specific because I'm like 99% positive this will change. I'm trying to name this for what you're really trying to do, which is waiting for the range to be "ready". I'll update the comment to be less specific about the leader lease.